stickers_bordure.py

```diff
@@ -4,31 +4,39 @@
 from gimpfu import pdb
 
 
-def stickerify_bordure(image, current_layer, black_grow=3, white_grow=12, shadow=True, canvas_increase=0, resize=False):
+def stickerify_bordure(image, current_layer, resize=False, black_grow=3, white_grow=12,
+                       border_color_fg=(1.0, 1.0, 1.0), border_color_bg=(0.0, 0.0, 0.0),
+                       shadow=True, canvas_increase=0):
     def duplicate_layer():
         copy = current_layer.copy()
         image.add_layer(copy)
         # copy is added above so we want to go down a bit
         image.active_layer = current_layer
         return copy
 
-    def fill_black():
+    def fill_bg():
         pdb.gimp_edit_bucket_fill(current_layer, 1, 0, 100, 255, 0, 0, 0)
 
-    def fill_white():
+    def fill_black():
```

Comment:
Is this function used anymore? It doesn't look like '-'

Reply:
Yes, fill_black is used to color the shadow. I prefered it this way because with some background colors (especially light ones), the shadow may seem a bit odd.